docs/source/commandline/commandline-all.rst

```diff
@@ -661,160 +1081,242 @@ Core input options
+
 **-[no]update_in_pause**
 
 	Enables updating of the main screen bitmap while the game is paused. This means that the VIDEO_UPDATE callback will be called repeatedly during pause, which can be useful for debugging. The default is OFF (*-noupdate_in_pause*).
 
+.. _mame-commandline-watchdog:
+
+**-watchdog** *<duration>* / **-wdog** *<duration>*
+
+	Enables an internal watchdog timer that will automatically kill the MAME process if more than *<duration>* seconds passes without a frame update. Keep in mind that some games sit for a while during load time without updating the screen, so *<duration>* should be long enough to cover that. 10-30 seconds on a modern system should be plenty in general. By default there is no watchdog.
+
 
-Core communication options
+Core Communication Options
 --------------------------
 
+.. _mame-commandline-commlocalhost:
+
 **-comm_localhost** *<string>*
 
 	Local address to bind to.  This can be a traditional xxx.xxx.xxx.xxx address or a string containing a resolvable hostname.  The default is value is "*0.0.0.0*"
 
+.. _mame-commandline-commlocalport:
+
 **-comm_localport** *<string>*
 
 	Local port to bind to.  This can be any traditional communications port as an unsigned 16-bit integer (0-65535).  The default value is "*15122*".
 
+.. _mame-commandline-commremotehost:
+
 **-comm_remotehost** *<string>*
 
 	Remote address to connect to.  This can be a traditional xxx.xxx.xxx.xxx address or a string containing a resolvable hostname.  The default is value is "*0.0.0.0*"
 
+.. _mame-commandline-commremoteport:
+
 **-comm_remoteport** *<string>*
 
 	Remote port to connect to.  This can be any traditional communications port as an unsigned 16-bit integer (0-65535).  The default value is "*15122*".
 
 
 
-Core misc options
+Core Misc Options
 -----------------
 
+.. _mame-commandline-drc:
+
 **-[no]drc**
 	Enable DRC cpu core if available.  The default is ON (*-drc*).
 
+.. _mame-commandline-drcusec:
+
 **\-drc_use_c**
 
 	Force DRC use the C code backend.  The default is OFF (*-nodrc_use_c*).
 
+.. _mame-commandline-drcloguml:
+
 **\-drc_log_uml**
 
 	Write DRC UML disassembly log.  The default is OFF (*-nodrc_log_uml*).
 
+.. _mame-commandline-drclognative:
+
 **\-drc_log_native**
 
 	write DRC native disassembly log.  The default is OFF (*-nodrc_log_native*).
 
+.. _mame-commandline-bios:
+
 **-bios** *<biosname>*
 
 	Specifies the specific BIOS to use with the current game, for game systems that make use of a BIOS. The **-listxml** output will list all of the possible BIOS names for a game. The default is '*default*'.
 
+.. _mame-commandline-cheat:
+
 **-[no]cheat** / **-[no]c**
 
 	Activates the cheat menu with autofire options and other tricks from the cheat database, if present. The default is OFF (*-nocheat*).
 
+.. _mame-commandline-skipgameinfo:
+
 **-[no]skip_gameinfo**
 
 	Forces MAME to skip displaying the game info screen. The default is OFF (*-noskip_gameinfo*).
 
+.. _mame-commandline-uifont:
+
```

Comment:
Don't explicitly reference TrueType - just say it an be a system font name.  It's OpenType as often as not these days.

Reply:
Fixed on my side.